phpcs.xml.dist

```diff
@@ -231,23 +231,18 @@
 	<rule ref="WordPress.Files.FileName">
 		<properties>
 			<property name="custom_test_class_whitelist" type="array">
-				<!-- Test case parent classes. -->
+				<!-- Test case parent classes in the "includes" folder, not yet accounted for in WPCS. -->
 				<element value="PHPUnit_Adapter_TestCase"/>
-				<element value="WP_UnitTestCase"/>
-				<element value="WP_Ajax_UnitTestCase"/>
-				<element value="WP_Canonical_UnitTestCase"/>
-				<element value="WP_Test_REST_TestCase"/>
-				<element value="WP_Test_REST_Controller_Testcase"/>
-				<element value="WP_Test_REST_Post_Type_Controller_Testcase"/>
-				<element value="WP_XMLRPC_UnitTestCase"/>
+				<element value="WP_Test_XML_TestCase"/>
+
+				<!-- Test case parent classes outside of the "includes" folder. -->
+				<element value="Tests_Query_Conditionals"/>
```

Comment:
Making a note for anyone reviewing:

Why is this one in the list? It's serving 2 roles:

- Test case as `Tests_Query_VerbosePageRules` extends it
- and a test class as it has tests within it